pkg/cloud/nodelocal/test_nodelocal_storage.go

```diff
@@ -49,17 +49,14 @@ func ReplaceNodeLocalForTesting(root string) func() {
 func TestingMakeNodelocalStorage(
 	root string, settings *cluster.Settings, es cloudpb.ExternalStorage,
 ) cloud.ExternalStorage {
-	if !buildutil.CrdbTestBuild {
```

Comment:
could you provide more context on this change? It seems unrelated to the rest of this commit.

Reply:
I removed this from the PR and may do something like this in the future if I add local support. 

The motivation for this change is the `roachtest` and `roachprod` binaries are not considered test binaries. So calling this would panic.